src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.html

```diff
@@ -162,15 +168,78 @@ <h5><span jhiTranslate="artemisApp.studentExamDetail.exercises">Exercises</span>
                                 *ngIf="
                                     exercise?.studentParticipations &&
                                         exercise?.studentParticipations.length > 0 &&
-                                        (exercise?.studentParticipations)![0].results &&
+                                        (exercise?.studentParticipations)![0].results?.length > 0 &&
                                         (exercise?.studentParticipations)![0].results[0].assessor;
                                     else noAssessor
                                 "
                             >
-                                {{ (exercise?.studentParticipations)![0].results[0].assessor.name }}
+                                {{ (exercise?.studentParticipations)![0].results[0].assessor?.name }}
                             </ng-container>
                             <ng-template #noAssessor></ng-template>
                         </td>
+                        <td class="align-middle">
```

Comment:
I think you shouldn't add td as well if you are not going to show anything.

Reply:
can you clarify that? because I am showing something there.